server/src/main/scala/net/wiringbits/repositories/models/Table.scala

```diff
@@ -0,0 +1,7 @@
+package net.wiringbits.repositories.models
+
+case class Table(
+    table_name: String
```

Comment:
`table_name` -> `tableName`, on the other hand, we know this is a table and we could just call it `name`.

I'm thinking that it could be worth naming the class like `DatabaseTable` to make it clear.